templates/resources/ovhcloud_connect_config_pop.md.tmpl

```diff
@@ -0,0 +1,32 @@
+---
+subcategory: "Ovhcloud Connect (OCC)"
+---
+
+# ovh_ovhcloud_connect_config_pop (Resource)
+
+Creates a pop configuration for an Ovhcloud Connect product.
+
+Please take a look at the list of available `types` in the `Required` section in order to know the list of available type configurations.
+
+## Example Usage
+
+{{tffile "examples/resources/ovhcloud_connect_config_pop/resource.tf"}}
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `interface_id` (Number) ID of the interface
+- `service_name` (String) Service name
+- `type` (String) Type of the pop configuration. Availaible types:
+    * `l2`
+    * `l3`
+
+### Optional
+
+- `customer_bgp_area` (Number) Customer Private AS
+- `id` (Number) ID of the Pop Configuration
+- `ovh_bgp_area` (Number) OVH Private AS
+- `status` (String) Status of the pop configuration
```

Comment:
the docs should be re-generated after the changes I commented on the `_gen.go` files (invalid `optional` set on computed-only attributes)